src/components/userProfile/components/UserInfo.js

```diff
@@ -16,30 +20,81 @@ const {
         TITLE, 
     },
     PROFILE_PIX,
-    STRINGS: { EDIT, GHOST, MEDIUM, PRIMARY, SMALL, SQUARE },
-    TEXTS: { EDIT_PHOTO, EDIT_PROFILE },
+    STRINGS: { 
+        ADDRESS,
+        DEFAULT, 
+        EMAIL, 
+        CITY, 
+        COUNTRY, 
+        EDIT, 
+        FIRST_NAME,  
+        GHOST, 
+        LAST_NAME, 
+        PHONE, 
+        PRIMARY, 
+        SMALL, 
+        SQUARE,
+        USERNAME, 
+    },
+    TEXTS: { BASIC_INFO_TEXT, CONTACT_INFO_TEXT, EDIT_PHOTO, EDIT_PROFILE, LOCATION_INFO_TEXT },
 } = USER_PAGE;
 
-// mapping various user information
-const profile = PROFILE_INFO.map(({ heading, info, key }) => ( 
-    <div key={key} className={INFO_DIV}>
-        <h4 className={HEADER_INFO}>
-            {heading}  
-        </h4>
-        {
-            info.map(({ description, title }) => (
-                <Row gutter={16} key={title} className={ROW_CONTAINER}>
-                    <Col span={12} className={TITLE}>{title}</Col>
-                    <Col span={12}>{description}</Col>
-                </Row>
-            ))
-        }
-    </div>
-));
-
 // react component used to render user information
-export default class UserInfo extends React.Component {
+class UserInfo extends React.Component {
+
     render() {
+
+        const { 
+            address, 
+            city, 
+            country, 
+            email, 
+            firstName,  
+            lastName, 
+            phoneNumber,
+            username, 
+        } = this.props.userData;
+
+        //This is the user information
+        const PROFILE_INFO = [{
+            heading: BASIC_INFO_TEXT,
+            info: [
+                { description: firstName, title: FIRST_NAME },
+                { description: lastName, title: LAST_NAME },
+                { description: username, title: USERNAME },  
+            ],
+        }, {
+            heading: CONTACT_INFO_TEXT,
+            info: [
+                { description: email,  title: EMAIL },
+                { description: phoneNumber,  title: PHONE },
+            ],
+        }, {
+            heading: LOCATION_INFO_TEXT,
+            info: [
+                { description: address, title: ADDRESS },
+                { description: city, title: CITY },
+                { description: country, title: COUNTRY }, 
+            ],
+        },
+        ];
```

Comment:
wrap the logic that generates `PROFILE_INFO` into a class method.
```
...
const profileInfo = this.generateProfileInfo();
...
```

Reply:
ok

Reply:
@enyeInc the file path could not be located
could not read getUserData of undefined

Reply:
seems like you have a pathing issue